### ISSUE TITLE
Re-enable continuous delivery

### ISSUE THREAD

**Issue by monperrus** [eclipse/repairnator#1167]
* [ ] add continuous delivery to Maven Central
* [ ] add continuous delivery to Dockerhub

**Comment by monperrus** [comment]
Dockerhub delivery is a problem because we cannot do Docker in Docker on Eclipse Jenkins.

**Comment by javierron** [comment]
In the Eclipse-Jenkins [docs](https://wiki.eclipse.org/Jenkins#I_want_to_build_a_custom_Docker_image_.28with_docker_build.29.2C_but_it_does_not_work._What_should_I_do.3F) they state that: _You cannot currently build images on the cluster (i.e. docker build does not work). We plan to address this shortcoming shortly._

**Comment by andre15silva** [comment]
We could ssh into a build farm as an alternative.

**Comment by ohbus** [comment]
How about we move this to GitHub Actions?

I will be willing to help you make this transformation.

**Comment by monperrus** [comment]
Yes, we can do CD with Github Actions. Do you want to give it a try?
